tools/gradle-plugins/gradle-curiostack-plugin/build.gradle.kts

```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation("com.github.ben-manes:gradle-versions-plugin")
     implementation("com.google.auth:google-auth-library-oauth2-http")
     implementation("com.google.cloud:google-cloud-kms")
+    implementation("com.google.cloud.tools:jib-build-plan:0.4.0")
```

Comment:
I guess they don't expose this yet since it's not 1.0. It means we'll continue to have tricky updates of jib